ParisBNF/abb/BNFabb94.xml

```diff
@@ -0,0 +1,324 @@
+                                </origin>
+                                <provenance>The manuscript was bought by
+                                  <persName ref="PRS1071dAbbadi" role="owner">Antoine d'Abbadie</persName>
+                                  sometime between <date>1840</date> and <date>1848</date>.
+                                </provenance>
+                        <acquisition/>
+                          </history>
+
+                          <additional>
+                             <adminInfo>
+                                <recordHist>
+                                   <source>
+                                      <listBibl type="catalogue">
+                                         <bibl>
+                                            <ptr target="bm:ContiRossini1914Notice"/>
+                                            <citedRange unit="page">182</citedRange>
+                                            <citedRange unit="number">170</citedRange>
+                                         </bibl>
+                                         <bibl>
+                                            <ptr target="bm:Chaine1912Catalogue"/>
+                                            <citedRange unit="page">60-61</citedRange>
+                                            <citedRange unit="number">94</citedRange>
+                                         </bibl>
+                                         <bibl>
+                                            <ptr target="bm:Abbadie1859Cat"/>
+                                            <citedRange unit="page">104-105</citedRange>
+                                            <citedRange unit="number">94</citedRange>
+                                         </bibl>
+                                           </listBibl>
+
+                                   </source>
+                                </recordHist>
+                             </adminInfo>
+                          </additional>
+
+                          <msPart xml:id="p1">
+                          <msIdentifier></msIdentifier>
+                          <msContents>
+                            <msItem xml:id="p1_i1">
+                                 <locus from="2r" to="20v"/>
+                                  <title type="complete" xml:lang="gez" ref="LIT1414GadlaA"/>
+                                  <textLang xml:lang="gez"></textLang>
+                                </msItem>
+
+                                <msItem xml:id="p1_i2">
+                                     <locus from="22r" to="32r"/>
+                                      <title type="complete" xml:lang="gez" ref="LIT5134KirosGeneral"/>
+                                      <textLang xml:lang="gez"></textLang>
+                                    </msItem>
+                                    </msContents>
+
+<physDesc>
+    
+    <objectDesc>
+        <layoutDesc>
+            <layout columns="2" writtenLines="19 24">
```

Comment:
In case one gives a different number of writtenLines than in Conti Rossini's catalogue, we also indicate which folio was considered.